common/components/chrome/MainHeader.jsx

```diff
@@ -53,6 +53,7 @@ class MainHeader extends React.PureComponent<{||}> {
     NavigationDispatcher.dispatch({
       type: 'SET_SECTION',
       section: Section.FindProjects,
+      page: 1,
```

Comment:
Why are we setting page on SET_SECTION?

Reply:
Missed that one. I don't think we need that anymore.